### PR TITLE
Register aiida-qp2 plugin

### PR DESCRIPTION
**Pull request by q-posev**

### CODE REVIEW

File: plugins.yaml [hunk 1]
```diff
@@ -436,3 +436,10 @@ aiida-zeopp:
   entry_point_prefix: zeopp
   pip_url: aiida-zeopp
   plugin_info: https://raw.github.com/ltalirz/aiida-zeopp/master/setup.json
+aiida-qp2:
+  code_home: https://github.com/TREX-CoE/aiida-qp2
+  development_status: beta
```
Comment (ltalirz) [inline plugins.yaml:441]: We've deprecated this variable and suggest you use the standard PyPI trove classifiers in the plugin (e.g. in the setup.json)
```suggestion
```

**Review by ltalirz — COMMENTED** [review]
thanks @q-posev - except for the development status this looks good;

let me know if it's urgent to merge this; if not I'll still wait a little bit until I've managed to get https://github.com/aiidateam/aiida-registry/pull/210 to run properly (which also fixes a minor issue in the registry website build)

File: plugins.yaml (hunk 1 above)
Comment (q-posev) [reply plugins.yaml:441]: Thank you @ltalirz ! Good point, I used the trove classifier but decided to duplicate it here. I will remove it.

**Comment by q-posev** [comment]
Thanks for the review @ltalirz ! 

There is no urgency to merge this. Just let me know if something is wrong on our side that breaks the build.

**Comment by q-posev** [comment]
Btw, I faced the following documentation build issue before with our `aiida-qp2` plugin:

```
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/markupsafe/__init__.py)
```

I fixed it by adding `pip install --force-reinstall MarkupSafe==2.0.1` to the CI

**Comment by ltalirz** [comment]
yeah, these guys made a bit of a mess; see recent commits on the plugin cutter which constraint the markupsafe version

**Comment by q-posev** [comment]
But then I do not understand why it is crashing here.

**Comment by ltalirz** [comment]
DOn't worry it's my fault, it's fixed in the other PR that I wanted to merge first

**Comment by ltalirz** [comment]
Sorry for the delay, was very busy

**Comment by q-posev** [comment]
No problem, thanks a lot @ltalirz !

**Pull request merged**